common/static/coffee/src/discussion/views/discussion_thread_view.coffee

```diff
@@ -22,6 +22,8 @@ if Backbone?
       if @mode not in ["tab", "inline"]
         throw new Error("invalid mode: " + @mode)
 
+      @readOnly = $(".discussion-module").data('read-only')
```

Comment:
Should we be adding the read-only data (with false) to the general inline discussion component templates?